tests/test_checker.py

```
@@ -0,0 +1,355 @@
+    def test_checker_anon_headers_detection(self, checker):
+        """Test detection of proxy-revealing headers."""
+        proxy_headers = [
+            'HTTP_X_FORWARDED_FOR',
+            'HTTP_VIA', 
+            'HTTP_PROXY_CONNECTION',
+            'HTTP_X_REAL_IP',
+            'HTTP_CLIENT_IP'
+        ]
+        
+        for header in proxy_headers:
+            headers = {header: 'some-value'}
```

Ruff (0.11.9)

Local variable `headers` is assigned to but never used

Remove assignment to unused variable `headers`

(F841)